src/base/projects/Browser.svelte

```diff
@@ -353,18 +392,18 @@
           {#await getAnchor}
             <Loading small margins />
           {:then anchor}
-            {#if anchor === commit}
+            {#if anchor?.includes(commit)}
               {#if commit === project.head}
                 <span class="anchor-widget anchor-latest">
                   <span class="anchor-label" title="{anchors}">anchored 🔒</span>
                 </span>
               {:else}
                 <span class="anchor-widget" on:click={() => navigateBrowser(project.head)}>
-                  <span class="anchor-label" title="{anchors}">anchored 🔒</span>
+                  <span class="anchor-label" title="{anchors}">{getAnchorIndexByCommit(anchor, commit)} anchor behind latest 🔒</span>
                 </span>
               {/if}
             {:else if anchor}
-              <span class="anchor-widget not-anchored" on:click={() => navigateBrowser(anchor)}>
+              <span class="anchor-widget not-anchored" on:click={() => navigateBrowser(anchor[0])}>
```

Comment:
How are the anchors sorted? Maybe worth sorting them explicitly in the GQL query.